### PR TITLE
refactor: move to typescript-react-apollo

### PR DESCRIPTION
**Pull request by bigint**

### CODE REVIEW

**Comment by bigint** [comment]
Kudos to @sasicodes

File: src/components/Settings/Account/SuperFollow.tsx [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 import { Spinner } from '@components/UI/Spinner';
 import useBroadcast from '@components/utils/hooks/useBroadcast';
-import type { Erc20, Mutation } from '@generated/types';
-import {
-  CreateSetFollowModuleTypedDataDocument,
-  EnabledCurrencyModulesWithProfileDocument
-} from '@generated/types';
+import type { Erc20 } from '@generated/types';
+import { useCreateSetFollowModuleTypedDataMutation } from '@generated/types';
+import { useEnabledCurrencyModulesWithProfileQuery } from '@generated/types';
```
Comment (harish-sethuraman) [inline src/components/Settings/Account/SuperFollow.tsx:11]: combine it

File: src/components/Settings/Profile/NFTPicture.tsx [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
-import type { Mutation, NftImage, Profile, UpdateProfileImageRequest } from '@generated/types';
-import {
-  CreateSetProfileImageUriTypedDataDocument,
-  CreateSetProfileImageUriViaDispatcherDocument,
-  NftChallengeDocument
-} from '@generated/types';
+import type { NftImage, Profile, UpdateProfileImageRequest } from '@generated/types';
+import { useCreateSetProfileImageUriViaDispatcherMutation } from '@generated/types';
+import { useCreateSetProfileImageUriTypedDataMutation } from '@generated/types';
+import { useNftChallengeLazyQuery } from '@generated/types';
```
Comment (harish-sethuraman) [inline src/components/Settings/Profile/NFTPicture.tsx:12]: Why are these not combined as well? 👀

File: src/components/Settings/Account/SuperFollow.tsx (hunk 1 above)
Comment (bigint) [reply src/components/Settings/Account/SuperFollow.tsx:11]: fixed

File: src/components/Settings/Profile/NFTPicture.tsx (hunk 2 above)
Comment (bigint) [reply src/components/Settings/Profile/NFTPicture.tsx]: fixed in https://github.com/lensterxyz/lenster/pull/1194/commits/200bc00f75fd6013c9616d353a1fa75940715478

not sure, need to find eslint rule for this 😕

**Pull request merged**